_posts/2019-10-15-dependency-management.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: blog-detail
+post-type: blog
+by: Alexandre Archambault, Sébastien Doeraene
+title: "Better management of transitive dependencies and conflicts"
+---
+
+Early in 2019, the Scala Center advisory board accepted [a proposal from Spotify](https://github.com/scalacenter/advisoryboard/blob/master/proposals/020-sbt-transitive-dependencies-conflicts.md), whose goal is to improve the handling of conflicts in dependency management in sbt.
+
+The proposal revolves around 3 axes:
+
+* improving the documentation about how versions are reconciled in sbt, and what to do about conflicts,
+* have a usable strict conflict manager in sbt, and
+* develop tools relying on static analysis to detect runtime conflicts between libraries that would not have been detected otherwise.
+
```

Comment:
This intro is publishable as-is, but I also think it could be improved to be clearer and more concrete. The verbiage in the bullets is pretty abstract.

Maybe something a little more vivid like:

> The work we've been doing may interest you if:
> * You've run into trouble with version conflicts between dependencies and didn't know what to do or where to look for advice.
> * You want more control how version conflicts are resolved in your build.
> * You want automated asisstance with deciding whether a version upgrade might break something in your project.

that kind of thing